packages/features/form-builder/FormBuilder.tsx

```diff
@@ -483,16 +488,36 @@ function FieldEditDialog({
                       }
                       label={t("identifier")}
                     />
-                    <InputField
-                      {...fieldForm.register("label")}
-                      // System fields have a defaultLabel, so there a label is not required
-                      required={
-                        !["system", "system-but-optional"].includes(fieldForm.getValues("editable") || "")
-                      }
-                      placeholder={t(fieldForm.getValues("defaultLabel") || "")}
-                      containerClassName="mt-6"
-                      label={t("label")}
-                    />
+                    <div>
+                      {fieldForm.getValues("type") === "boolean" ? (
```

Comment:
Don't we have `fieldForm.getValues("type")` saved in some variable?

Reply:
Not currently, but I’ve added it as it is used many times